hazelcast/src/main/java/com/hazelcast/map/impl/MapMigrationAwareService.java

```diff
@@ -73,8 +73,27 @@ public boolean isKnownServiceNamespace(ServiceNamespace namespace) {
 
     @Override
     public void beforeMigration(PartitionMigrationEvent event) {
+        flushAndRemoveQueryCaches(event);
```

Comment:
Since we are now flushing the query caches in `beforeMigration`, shouldn't the same code fragment be removed from `commitMigration` like we did in the patch to `master`?

Reply:
Good catch, it was a left over. Removed it.